apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -370,6 +397,12 @@ def _get_nrf_device_args(options):
         type="bool",
         help="Run on the FVP emulator instead of hardware.",
     ),
+    server.ProjectOption(
+        "heap_size",
+        optional=["generate_project"],
+        type="int",
+        help="Sets HEAP_SIZE for Zephyr board.",
```

Comment:
since HEAP_SIZE isn't a Zephyr option, can you instead say something more like "Sets the size of the heap used to service TVM memory allocation requests"

Reply:
Or maybe something more direct like "Sets the value for HEAP_SIZE passed to K_HEAP_DEFINE()" since K_HEAP_DEFINE is a zephyr macro.